deploy/scripts/src/ipv-core/test.ts

```diff
@@ -189,7 +189,7 @@ if (!validEnvironments.includes(environment))
 const env = {
   orchStubEndPoint: __ENV[`IDENTITY_${environment}_ORCH_STUB_URL`],
   ipvCoreURL: __ENV[`IDENTITY_${environment}_CORE_URL`],
-  vtrText: __ENV[`IDENTITY_${environment}_VTR_TEXT`]
+  vtrText: __ENV[`IDENTITY_${environment}_CORE_VTR_TEXT`]
```

Comment:
This new variable is not being used anywhere. Is there a matching line missing?

Reply:
This is not a new variable. I have just updated the name of the env var to keep it in line with the naming convention that we are following. The object property `env.vtrText` is being used in Line 233

``/authorize?journeyType=full&userIdText=${userId}&signInJourneyIdText=${signInJourneyId}&vtrText=${env.vtrText}&targetEnvironment=${environment}&reproveIdentity=NOT_PRESENT&emailAddress=${testEmail}&votText=&jsonPayload=&evidenceJsonPayload=&error=recoverable`,`

Reply:
Right you are